code/Test_definitions/Traffic_Influence.yaml

```diff
@@ -0,0 +1,1030 @@
+    The RESTful TI API endpoint, for example
+    [**https://tim-api.developer.tim.it/trafficinfluence**](https://tim-api.\
+    developer.tim.it/trafficinfluence)\
+    \
+    **TrafficInfluence:**
+    This object represents the resource that carries the requirements from the
+    user to be implemented. The TI API is invoked for the life cycle management
+    of this resource (CRUD). The resource contains the intents from the TI API
+    Consumer. Managing this resource, the developer can specify in which
+    geographical location the routing must be applied, toward which application,
+    maybe for a specific set of users or for a limited period of time.\
+    \
+    **trafficInfluenceID:**
+    Identifier for the Traffic Influence resource. This parameter is returned
+    by the TI API and must be used to update it (e.g., adding a Device or
+    deleting it). A different Traffic Influence resource must be created for
+    any Device or Zone or Region. All these resources are related to an
+    Application identified by "appId".\
+    \
+    **apiConsumerId:**
+    Unique identifier for the TI API Consumer.\
+    \
+    **region:**
+    The developer can specify in which geographical area he requires the fastest
+    routing toward the nearest application instance. A "region" is a wide
+    geographical area and it contains one or more "zones". A "zone" is where the
+    Edge Cloud Zone is located. Zones and Regions identifiers are defined and
+    provided by the Telco Operator and can also be used or retrieved with other
+    CAMARA APIs (“MEC Exposure & Experience Management API” and “Simple Edge
+    Discovery”). To add more regions the TI API must be invoked (POST) for each
+    region. New "TrafficInfluence" resources are created (with different
+    "trafficInfluenceID"). All the created resources are aggregated by the
+    Application (identified by "appId").\
+    \
+    **zone:**
+    The developer can specify in which geographical area he requires the fastest
+    routing toward the nearest Application instance. A "zone" is a smaller
+    geographical area inside a "region". A "zone" is where the Edge Cloud Zone
+    is located.\
+    To add more zones the TI API must be invoked (POST) for each "zone".
+    New "TrafficInfluence" resources are created (with different
+    "trafficInfluenceID"). All the created resources are aggregated by the
+    Application (identified by "appId").\
+    \
+    **appId:**
+    A globally unique identifier associated with the application. This
+    identifier is provided during the application onboarding process.
+    To influence the traffic toward a specific Application. It is the Operator
+    Platform that detects the appropriate Application instance in the selected
+    "region" or "zone".\
+    \
+    **appInstanceId:**
+    A globally unique identifier generated by the Operator Platform to identify
+    a specific instance of the Application on a specific zone. To influence a
```

Comment:
Here is another related aspect of zone and region. The App instance id is referred to be associated to a zone but then if someone want to use region then how the instance id can be determined for the region?